mass_editing/models/mass_object.py

```diff
@@ -96,43 +100,41 @@ def create_action(self, cr, uid, ids, context=None):
                         "ir.actions.act_window," +
                         str(vals['ref_ir_act_window'])),
                     'object': True,
-                },
-                context)
+                }).id
+        _logger.debug("VALS")
+        _logger.debug(vals)
         self.write(
-            cr, uid, ids,
             {
                 'ref_ir_act_window': vals.get('ref_ir_act_window', False),
                 'ref_ir_value': vals.get('ref_ir_value', False),
-            },
-            context)
+            })
         return True
 
-    def unlink_action(self, cr, uid, ids, context=None):
-        for template in self.browse(cr, uid, ids, context=context):
+    @api.multi
+    def unlink_action(self):
+        for template in self:
+            template = self
```

Comment:
this line assigns the original browse record again, which fails below for len(self) > 1. remove